lib/h-include.js

```diff
@@ -133,7 +144,16 @@ window.HInclude.HIncludeElement = window.HIncludeElement = (function() {
 
     outstanding += 1;
     req.onreadystatechange = function () {
-      includeCallback(element, req);
+      if (req.readyState === 4) {
+        // Predicate for an alternative src should a request error occur
+        if(req.status !== 200 && req.status !== 304 && altUrl && !altSrcInclude) {
+          altSrcInclude = true;
+          // Run altSrcInclude pass with alternative url 
+          include(element, includeCallback);
```

Comment:
Could you send altSrcInclude as a function parameter to include, instead of having an object variable?

Reply:
I thought about doing it that way but as the latest refactor resulted in more methods, it would mean a lot of param pushing across all methods in order to use this variable.